File: toolbar-title/app/src/main/java/com/sqisland/espresso/toolbar_title/MainActivity.java
package com.sqisland.espresso.toolbar_title;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setTitle(R.string.my_title);
  }
}

File: toolbar-title/app/src/main/java/com/sqisland/espresso/toolbar_title/MainActivity.kt
package com.sqisland.espresso.toolbar_title

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setTitle(R.string.my_title)
  }
}